Sources/TSCBasic/TerminalController.swift

```diff
@@ -109,11 +109,15 @@ public final class TerminalController {
 
     /// Computes the terminal type of the stream.
     public static func terminalType(_ stream: LocalFileOutputByteStream) -> TerminalType {
+#if os(Windows)
+        return _isatty(_fileno(stream.filePointer)) == 0 ? .file : .tty
+#else
         if ProcessEnv.vars["TERM"] == "dumb" {
```

Comment:
this not needed on windows?

Reply:
No, there is no terminal emulator environment variable on Windows.  You cannot query whether the current emulator is a dumb terminal or not because there never was the concept of a dumb terminal for Windows.  It is a purely graphical environment, and so the only terminal emulator that is available is guaranteed to not be a dumb terminal.  That said, using that as an approximation of whether the terminal emulator supports extended ASCII codes or not is not really viable on Windows.  You do not have a way to query that and the best you can do is assume that the terminal emulator understands the extended VT220 ANSI escape codes (which it does if it is Windows Terminal) and let the terminal display the sequence if it does not (e.g. it is the old ConHost.exe process hosting the Windows console).